FreeRTOS-Plus/Demo/FreeRTOS_Plus_WolfSSL_FIPS_Ready_Windows_Simulator/README_wolfSSL_FIPS_ready.md

```diff
@@ -36,6 +36,19 @@ All required settings have been set in the user_settings.h header file included
 2. Run the RTOSDemo.exe 
 
 You will see a console that pops up, and it shows output like the following:
+<<<<<<< HEAD
+
+```
+Starting Power On Self Test
+In core integrity check error: hash = C66491A040D5B9686BAA7A75A280290D91B49...
+ERROR: -203
+Power On Self Test(Known-Answer-Test) FAILURE
+```
+
+Do not warry about this result, an error is expected at this point. Error number "-203" means In-Core-integrity-check failed. The check is identical to the "In Core Integrity Test" listed in the previous section. And the subsequent KAT also failed due to the first error. Once FIPS Ready has failed POST, it enters an error state and never allows subsequent cryptographic operations until the device is restarted and the tests can complete successfully. 

```

Comment:
Spelling - Warry -> worry

Reply:
This misspelling has gone with above cleanup.